product_docs/docs/pgd/5/manual/03-configuring-repositories.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Configuring PGD repositories
+deepToC: true
+---
+
+To install and run PGD requires that you configure repositories so that the system can download the appropriate packages.
+
+The following operations should be carried out on each host you wish to be a data node, witness node or subscribe-only node.
+
+For our example, that means configuring the repositories on all three hosts.
+
+### EDB account
+
+You'll need an EDB account to install Postgres Distributed. 
+
+You should have an EDB account. Use this to sign in to the [EDB Repos 2.0](https://www.enterprisedb.com/repos-downloads) page, where you can obtain your repo token. 
+
+On your first visit to this page, select **Request Access** to generate your repo token. Copy the token using the **Copy Token** icon, and store it safely. 
+
+### Setting environment variables
+
+First, set the `EDB_SUBSCRIPTION_TOKEN` environment variable to the value of your EDB repo token, obtained in the [EDB account](#edb-account) step.
+
+```
+export EDB_SUBSCRIPTION_TOKEN=<your-repo-token>
+```
+
+You can add this to your `.bashrc` script or similar shell profile to ensure it's always set. 
+
+!!! Note 
+Your preferred platform may support storing this variable as a secret which can appear as an environment variable. If this is the case, don't add the setting to `.bashrc` and instead add it to your platform's secret managent.
+!!!
+
+### Configure the repository
+
+All the software you need is available from the EDB Postgres Distributed package repository.
+You have the option to simply download and run a script to configure the EDB Postgres Distributed repository. 
+You can also download, inspect and then run that same script.
+The following instructions also include the essential steps that the scripts take for any user wanting to manually run, or automate, the installation process.
+
+
+#### RHEL/Other RHEL-based
+
+You can autoinstall with automated OS detection
+
+```
+curl -1sLf "https://downloads.enterprisedb.com/$EDB_SUBSCRIPTION_TOKEN/postgres_distributed/setup.rpm.sh" | sudo -E bash
```

Comment:
The former is when we are actively substituting in a token from the environment. 
The latter is when we are obscuring a token from the repos generated output.

To make them the same would potentially lead to more, rather than less, confusion.

Reply:
Reworked anyway to make the obscuring element more obvious and rely on the upfront explanation.